### PR TITLE
Fix styling of cohort panel buttons, update dependencies. 

### PR DESCRIPTION
**Pull request by ewelinagr**
Fixes [PMCSD-16](https://thehyve.atlassian.net/browse/PMCSD-16).

### CODE REVIEW

File: src/app/modules/gb-side-panel-module/accordion-components/gb-cohorts/gb-cohorts.component.css [hunk 1]
```diff
@@ -149,11 +149,18 @@ hr {
...
+.panel-header-btn-block {
+  margin-right: 0.5em;
+}
+
 mat-expansion-panel-header button, .ui-button-secondary {
   margin-top: auto;
   color: #505050;
   border: none;
   background-color: transparent;
+  padding-left: 0.4em;
```
Comment (gijskant) [inline src/app/modules/gb-side-panel-module/accordion-components/gb-cohorts/gb-cohorts.component.css:162]: Maybe even a bit more, say `0.6em`?

File: src/app/modules/gb-side-panel-module/accordion-components/gb-cohorts/gb-cohorts.component.css [hunk 2]
```diff
@@ -167,9 +174,7 @@ mat-expansion-panel mat-expansion-panel-header {
   color: #606060 !important;
   background-color: transparent !important;
   border: black;
-  padding: 0em;
-  margin: 0px -3px 0px 0px;
-  -moz-margin-start: 1.1em;
+  padding: 0 0.4em 0 0.4em;
```
Comment (gijskant) [inline src/app/modules/gb-side-panel-module/accordion-components/gb-cohorts/gb-cohorts.component.css:177]: Maybe even `0.6em`?

**Review by gijskant — APPROVED** [review]
Yes, this fixes it, nice! However, I think it looks a little bit better with slightly more padding. Do you agree? The current padding is also acceptable for me.

**Comment by ewelinagr** [comment]
> Yes, this fixes it, nice! However, I think it looks a little bit better with slightly more padding. Do you agree? The current padding is also acceptable for me.

You're right, I increased the padding.

**Pull request merged**